plugins/CoreHome/vue/src/Notification/Notification.less

```diff
@@ -8,6 +8,10 @@
     margin-top: 1em;
   }
 
```

Comment:
not sure if we need a `max-width:1300px` here. see the image, the notification is not lined up on a large screen.
![image](https://user-images.githubusercontent.com/3783072/141721059-4dc5743f-c366-4f18-a521-895c89102479.png)

Reply:
@peterhashair this looks strange, I agree, but I think it's a problem w/ admin pages, since it looks fine on the dashboard and other report pages (the widgets stretch as far as the notification). I checked on 4.x-dev and it's an issue there so I'm guessing this might be a good candidate for a new issue :+1: